cmd/flags/auth.go

```diff
@@ -178,3 +211,42 @@ func issuerCertificates(vipr *viper.Viper) ([]*x509.Certificate, error) {
 
 	return certs, nil
 }
+
+func apiKeyCsvFile(vipr *viper.Viper) (map[string]*apikey.UserNameAndGroups, error) {
+	// Open the file
+	csvFileName := vipr.GetString(authAPIKeyFileViperKey)
+	csvfile, err := os.Open(csvFileName)
+	if err != nil {
+		return nil, err
+	}
+	defer func(csvfile *os.File) {
+		_ = csvfile.Close()
+	}(csvfile)
+
+	// Parse the file
+	r := csv.NewReader(csvfile)
+	// Set separator as ";"
+	r.Comma = ';'
```

Comment:
could you make separator char and comment chat a constant so it is easier to locate for future changes?